docs/README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Do we need this empty line? It probably causes that change. Remove if not needed.  
<img width="1134" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/9ef880e6-fffa-4410-82dc-d078800ea0b0">

Reply:
Removed unneeded spaces.